modules/transaction-payment/src/lib.rs

```diff
@@ -355,19 +364,33 @@ pub mod module {
 				);
 			})
 		}
+
+		// remove after runtime upgraded
+		fn on_runtime_upgrade() -> Weight {
+			DefaultFeeCurrencyId::<T>::remove_all(None);
+			T::DbWeight::get().reads_writes(0, 1)
```

Comment:
we need to check how many items we have currently and change writes.

Reply:
There are currently 34 items in total, which is an easy mission for upgrades.